static/js/services/enketo.js

```diff
@@ -272,35 +276,30 @@ angular.module('inboxServices').service('Enketo',
         });
     };
 
-    var xmlToDocs = function(doc, record) {
-      var docsToStore = [], recordDoc, $record;
+    var xmlToDocs = function(doc, formInternalId, record) {
 
       function mapOrAssignId(e, id) {
-        var $e, $id;
-
         if (!id) {
-          $e = $(e);
-          $id = $e.children('_id');
-
+          var $id = $(e).children('_id');
```

Comment:
I've just removed the split declaration, and inlined the variable that was only used here.